src/main/scala/com/stratio/hermes/utils/Hermes.scala

```diff
@@ -16,55 +16,155 @@
 
 package com.stratio.hermes.utils
 
+import java.security.InvalidParameterException
+
 import com.stratio.hermes.constants.HermesConstants
-import com.stratio.hermes.helpers.RandomHelper
 import com.stratio.hermes.implicits.HermesSerializer
 import com.stratio.hermes.models.NameModel
 import org.json4s._
 import org.json4s.native.Serialization.read
-
-import scala.util.Try
+import com.stratio.hermes.helpers.RandomHelper
+import scala.language.postfixOps
+import scala.util.{Random, Try}
 
 /**
- * Hermes util used for to generate random values.
- */
+  * Hermes util used for to generate random values.
+  */
 case class Hermes(locale: String = HermesConstants.ConstantDefaultLocale) extends HermesSerializer {
 
   /**
-   * Generates random names.
-   */
+    * Generates random names.
+    */
   object Name extends HermesUnit {
 
     override def unitName(): String = "name"
 
     lazy val nameModel =
-      Try(read[NameModel] (getClass.getResourceAsStream(s"/locales/$unitName/$locale.json")))
+      Try(read[NameModel](getClass.getResourceAsStream(s"/locales/$unitName/$locale.json")))
         .getOrElse(throw new IllegalStateException(s"Error loading locale: /locales/$unitName/$locale.json"))
 
     /**
-     * Example: "Bruce Wayne".
-     * @return a full name.
-     */
-    def fullName() : String = s"$firstName $lastName"
+      * Example: "Bruce Wayne".
+      *
+      * @return a full name.
+      */
+    def fullName(): String = s"$firstName $lastName"
 
     /**
-     * Example: "Bruce Lex".
-     * @return a middle name.
-     */
-    def middleName() : String = s"$firstName $firstName"
+      * Example: "Bruce Lex".
+      *
+      * @return a middle name.
+      */
+    def middleName(): String = s"$firstName $firstName"
 
     /**
-     * Example: "Bruce".
-     * @return a first name.
-     */
-    def firstName() : String =
+      * Example: "Bruce".
+      *
+      * @return a first name.
+      */
+    def firstName(): String =
       RandomHelper.randomElementFromAList[String](nameModel.firstNames).getOrElse(throw new NoSuchElementException)
 
     /**
-     * Example: "Wayne".
-     * @return a last name.
-     */
-    def lastName() : String =
+      * Example: "Wayne".
+      *
+      * @return a last name.
+      */
+    def lastName(): String =
       RandomHelper.randomElementFromAList[String](nameModel.lastNames).getOrElse(throw new NoSuchElementException)
   }
+
+  /**
+    * Generates random numbers.
+    */
+  object NumberGenerator {
+
+    /**
+      * Example: "number(3) 123".
+      *
+      * @param n integer size.
+      * @return a random integer.
+      */
+    def number(n: Int): Int = {
+      assert(n>=0 && n<=9,throw new InvalidParameterException(s"$n must be between 0 and 9"))
+      if (n == 0) {
+        0
+      }
```

Comment:
if {
} else {
}

You have

if {
}
else {
}

The same to all ifs of your PR